django_freeradius/base/admin_filters.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.admin import SimpleListFilter
+from django.db.models import Count
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+
+class DuplicateListFilter(SimpleListFilter):
+    title = _('find duplicates')
+    parameter_name = 'duplicates'
+
+    def lookups(self, request, model_admin):
+        return (
+                ('username', _('username')),
+                ('value', _('value')),
+               )
+
+    def queryset(self, request, queryset):
```

Comment:
what about putting this logic in a model manager and calling the manager from this method?

That way we can reuse this logic in other views (eg: API) + it's easier to unit-test (models are easier to test)

Reply:
Yes I will, good hint!

Reply:
done